poc_iot_verifier/src/main.rs

```diff
@@ -67,15 +70,62 @@ impl Server {
         });
 
         let gateway_cache = GatewayCache::from_settings(settings).await?;
+
+        let (file_upload_tx, file_upload_rx) = file_upload::message_channel();
+        let file_upload =
+            file_upload::FileUpload::from_settings(&settings.output, file_upload_rx).await?;
+
+        let store_base_path = std::path::Path::new(&settings.cache);
+        // Gateway reward shares sink
+        let (gateway_rewards_tx, gateway_rewards_rx) = file_sink::message_channel(50);
+        let mut gateway_rewards = file_sink::FileSinkBuilder::new(
+            FileType::GatewayRewardShare,
+            store_base_path,
+            gateway_rewards_rx,
+        )
+        .deposits(Some(file_upload_tx.clone()))
+        .write_manifest(true)
+        .create()
+        .await?;
+
+        // Reward manifest
+        let (reward_manifest_tx, reward_manifest_rx) = file_sink::message_channel(50);
+        let mut reward_manifests = file_sink::FileSinkBuilder::new(
+            FileType::RewardManifest,
+            store_base_path,
+            reward_manifest_rx,
+        )
+        .deposits(Some(file_upload_tx.clone()))
+        .roll_time(Duration::minutes(15))
```

Comment:
you might want to make the roll time for this shorter for testing

Reply:
yeah, probably a good call